specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getQuotaRequestStatusById.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus",
+    "id": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/serviceLimitsRequests",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "properties": {
+          "requestSubmitTime": "2019-08-19T19:23:17.904Z",
+          "message": "Request completed",
+          "provisioningState": "Succeeded",
+          "value": [
+            {
+              "limit": 200,
+              "subRequestId": "AD07450A-DE86-4FD3-859B-107BEF218C4C",
+              "name": {
+                "value": "standardHCSFamily",
+                "localizedValue": "Standard HCS Family vCPUs"
+              },
+              "message": "Request completed",
+              "provisioningState": "Succeeded"
+            },
+            {
+              "limit": 50,
+              "name": {
+                "value": "standardNCPromoFamily",
+                "localizedValue": "Standard NC Promo Family vCPUs"
+              },
+              "message": "Request completed",
+              "provisioningState": "Succeeded"
+            }
```

Comment:
PUT is for a single resource only. Is the same request ID shared by multiple PUTs?

Reply:
No, each request id is unique.

Reply:
Then how a single request can have an array of quotaLimits resources? (i.e. you cannot update multiple quotaLimits resources by a single PUT)